src/tests/deployHandler.test.ts

```diff
@@ -0,0 +1,63 @@
+import { beforeEach, beforeAll, describe, expect, it, vi } from 'vitest';
+import * as keys from '../lib/keys.js';
+import * as file from '../lib/file.js';
+import * as prompts from '../lib/prompts.js';
+import * as web3 from '../lib/web3.js';
+import * as constants from '../lib/constants.js';
+
+import { Network } from 'src/types/index.js';
+import TestSetup from '../tests/mocks/TestSetup.json';
+import { deployHandler } from '~/commands/deploy/deployHandler.js';
+
+import {config} from "dotenv"
```

Comment:
'dotenv' should be in single quotes like the rest of the imports for consistency